include/swift/AST/DiagnosticsSema.def

```diff
@@ -2446,6 +2446,12 @@ WARNING(differentiable_implicit_noderivative_fixit,none,
       "stored property %0 has no derivative because it does not conform to "
```

Comment:
Can you rename this to `differentiable_nondiff_type_implicit_noderivative_fixit`?

Reply:
Done in https://github.com/apple/swift/pull/22905/commits/eb580db59d15b7dbe98178f350d680a59846a95b.